cmake/std/atdm/README.md

```diff
@@ -492,14 +492,18 @@ options are read.
 
 The specific `cmake/std/atdm/<system-name>/` sub-directories and the systems
 they support are:
+* `chama/`: Supports SNL HPC machine `chama/`.
```

Comment:
@fryeguy52, can we take the `/` off of the end of "Supports SNL HPC Machine `chama/`"?  The machine name does not have a `/`, just the system directory.